[TASK] Remove all references to TYPO3 v9 and below (#2430)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Autoloading/Background.rst

Instead of looking up every single class and caching the information
away, composer works on a “prefix”-based resolution. As an example, the
Composer class loader only needs to know that all PHP classes starting

> with :php:`TYPO3\CMS\Core` are located within :file:`vendor/typo3/cms-core/Classes`

Reviewer: Please use "EXT:core/Classes" as you want to backport this PR to v11. "vendor/typo3/cms-core" is only true for users with typo3/cms-composer-installers v4 (which is an absolute minority).

> with :php:`TYPO3\CMS\Core` are located within :file:`vendor/typo3/cms-core/Classes`
> (:file:`typo3/sysext/core/Classes` in legacy installations). The
> rest is done by a simple resolution to include the necessary PHP class

Author: ```suggestion
with :php:`TYPO3\CMS\Core` are located within :file:`EXT:core/Classes`. The
rest is done by a simple resolution to include the necessary PHP class
```

files. This means that the information to be cached away is only the

> list of available namespace-prefixes.

Reviewer: ```suggestion
list of available namespace prefixes.
```

> The definition of these prefixes is set inside the :file:`composer.json` of each

Reviewer: ```suggestion
The definition of these prefixes is set inside the :file:`composer.json` file of each
```
